github-actions/verify-pr-title/src/common.ts

```diff
@@ -11,5 +11,5 @@ export const WORK_ITEM_PORTION = "@W-\\d{4,9}@"
  * colon, or semicolon characters. All pull request titles allow these
  * characters as separators between portions
  */
-export const SEPARATOR = "[ -.,;:]*";
+export const SEPARATOR = "[\\s\\-\\.\\,\\;\\:]*";
```

Comment:
The online regex tester I used when I was playing around with this didn't like the unescaped special characters, so I just went ahead and escaped them.